group_vars/artemistests_common_config.yml

```diff
@@ -24,6 +24,8 @@ artemis_ssh_priv_key_value: ""
 use_docker: true
 artemis_system_ram_proportion: 0.6 # Limit Artemis to 60% RAM (also runs DB on same VM)
 
+apollon_url: "https://apollon.ase.in.tum.de/api/converter"
```

Comment:
Please ensure that you use the recent url (ase.cit or aet.cit). 
@FelixTJDietrich What is the current URL?